src/maps/factory.ts

```diff
@@ -132,17 +132,28 @@ export function town(townName: string): LandData {
 
 export function interCityConnections(
   grid: GridData,
-  connections: string[][],
+  connections: Array<[string, string, number?]>,
```

Comment:
At this point, we might want to just change it to a struct so that way we don't have to follow some magic pattern.

Something like:
interface Connection {
  connects: [string, string];
  cost?: number;
}

Reply:
Yea, I was trying to stay backwards compatible, but I can refactor this.